fastatophy/fastatophy.go

```diff
@@ -20,55 +19,89 @@ import (
 	"github.com/biogo/biogo/seq/linear"
 )
 
+// padding returns requested number of spaces as a
+// concatenated string.
+func padding(n int) (spaces string) {
```

Comment:
A much simpler way of writing this is 

```
const padding = "          "
```

and move it down to where it is used.